Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -71,6 +75,39 @@ const CreateNewMaintenanceWindow = () => {
     }));
   };
 
+  const handleSubmit = async () => {
+    const data = {
+      repeat: values.repeat,
+      date: values.date.format("YYYY-MM-DD"),
+      startTime: values.startTime.format("HH:mm"),
+      duration: values.duration,
```

Comment:
If the input here can only be numbers we should restrict the input type to that?  Doesn't make sense to allow the user to enter non-numerical characters if they're going to be rejected by vaildation anyway :thinking:

Reply:
Done ✅

![Screenshot 2024-08-21 224312](https://github.com/user-attachments/assets/eaf31a1d-878d-4690-ba3e-83cc86f93d07)